hedera-node/hedera-app/src/main/java/com/hedera/node/app/throttle/impl/NetworkUtilizationManagerImpl.java

```diff
@@ -201,4 +201,9 @@ public boolean wasLastTxnGasThrottled() {
     public void leakUnusedGasPreviouslyReserved(@NonNull final TransactionInfo txnInfo, long value) {
         backendThrottle.leakUnusedGasPreviouslyReserved(txnInfo, value);
     }
+
+    public boolean shouldThrottle(@NonNull TransactionInfo txnInfo, HederaState state) {
+        final var now = Instant.now();
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**issue:** ‏Using `Instant.now()` doesn't seem right, in order to be deterministic we should be using consensus time in the backend throttle (and during handle in general). In mono we are passing it [here](https://github.com/hashgraph/hedera-services/blob/5a68acf8bdda2a8656104490d774bfa4afd20223/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/throttling/TxnAwareHandleThrottling.java#L45).